src/node.js

```diff
@@ -159,8 +159,26 @@
   startup.globalVariables = function() {
     global.process = process;
     global.global = global;
-    global.GLOBAL = global;
-    global.root = global;
+
+    // Deprecate GLOBAL and root
+    ['GLOBAL', 'root'].forEach(function(name) {
+      // getter
+      const get = NativeModule.require('util').deprecate(function() {
+        return this;
+      }, "'" + name + "' is deprecated, use 'global'");
+      // setter
+      const set = function(value) {
+        const desc = {configurable: true, enumerable: true, value: value};
```

Comment:
Hmm, I think it's highly unlikely someone would use both in the same code, but if it's possible, go for it.